go.mod

```diff
@@ -74,12 +40,71 @@ exclude (
 	k8s.io/sample-apiserver v0.0.0
 )
 
-// Invalid pseudo-version.
-exclude github.com/go-check/check v1.0.0-20180628173108-788fd7840127
+// TEMPORARY HACK, DO NOT MERGE THIS
+replace github.com/datawire/go-mkopensource => github.com/LukeShu/go-mkopensource v0.0.0-20250206080114-4ff6b660d8d4
```

Comment:
lol

If a conversation about the maintainership of go-mkopensource hasn't happened yet, I'd suggest just forking it into the emissary-ingress org. But that's mostly-unrelated to this PR.

Reply:
Yeaaaaah that conversation is happening shortly. 🙂